.envExample

```diff
@@ -1,7 +1,11 @@
-PORT=localhost:8081
+PORT=127.0.0.1:8089
 HTTPS=false
 HTTPS_PORT=4000
 SESSION_KEY=YourSessionKey
 NODE_ENV=development
-BASE_URL=http://localhost:8081
-FILE_SIZE_UPLOAD_LIMIT_MB=512
\ No newline at end of file
+BASE_URL=http://127.0.0.1:8089
+FILE_SIZE_UPLOAD_LIMIT_MB=512
+ALLOWED_FILES=wad,pk3,zip
+ALLOWED_HEADERS=PWAD,PK,PK
```

Comment:
is `PK,PK` the same? (also it's not readable)

Reply:
it's a mapping 
```
ALLOWED_FILES=wad,pk3,zip
ALLOWED_HEADERS=PWAD,PK,PK
```
`WAD` = `PWAD`
`pk3` = `PK`
`zip` = `PK`

zip and pk3 are the same